frontend/src/lib/components/PersonalAPIKeys/PersonalAPIKeys.tsx

```diff
@@ -157,14 +158,14 @@ export function PersonalAPIKeys(): JSX.Element {
                     More about API authentication in PostHog Docs.
                 </a>
             </p>
-            <Button
+            <LemonButton
```

Comment:
I would actually hold off on moving these over, as `LemonButton`s are quite chunkier than Ant `Input`s. This should me migrated to the new style all at once.

Reply:
Oh, I see the input just was made lemon too 😅 But now there's a visible mismatch between "Details" and "Change Password"

Reply:
To my eye it looks **very** odd if one button is different from the others but not really that odd at all if the password inputs are slightly thinner.

Reply:
<img width="751" alt="Screenshot 2022-05-10 at 11 55 51" src="https://user-images.githubusercontent.com/2536520/167602378-3aeee719-345a-4b82-830d-d738163813c6.png">